userbenchmark/optim/regression_detector.py

```diff
@@ -2,6 +2,7 @@
 from ..utils import TorchBenchABTestResult, TorchBenchABTestMetric
 
 DEFAULT_REGRESSION_DELTA_THRESHOLD = 0.3
+COMPILE_TIME_REGRESSION_DELTA_THRESHOLD = 2.0
```

Comment:
compile time is a lot more fickle cuz we run it once only (compilation is slowww)